openpype/modules/ftrack/tray/ftrack_tray.py

```diff
@@ -48,6 +53,45 @@ def show_login_widget(self):
         self.widget_login.activateWindow()
         self.widget_login.raise_()
 
+    def show_ftrack_browser(self):
+        env_pf64 = os.environ['ProgramW6432'].replace(
+            ntpath.sep, posixpath.sep)
+        env_pf32 = os.environ['ProgramFiles(x86)'].replace(
+            ntpath.sep, posixpath.sep)
+        env_loc = os.environ['LocalAppData'].replace(
+            ntpath.sep, posixpath.sep)
+        chromium_paths_win = [
```

Comment:
I don't think you need all this stuff. You are still trying to force paths you define. User can have it installed, but is not using it as main browser. If I would be on windows without chrome but with firefox it would open MicrosoftEdge -> not something I want to...

All browsers I've looked for have `--app` argument available. So this could be simplified to:
```
url_arg = self.module.ftrack_url
if self.module.ftrack_open_as_app:
    url_arg = f"--app={self.module.ftrack_url}"
webbrowser.get().open_new(url_arg)
```

Which means: Get default browser and open the argument I've passed.